### DISCUSSION TITLE
Remover as Listas de figuras e tabelas do Indíce

### DISCUSSION THREAD

**Discussion by nunes3000** [joaomlourenco/novathesis#202] [Q&A] [answered]
Seria possível omitir a lista de figuras e de tabelas do Índice, e ao mesmo tempo continuarem a existir no corpo do documento tal como ilustrado na seguinte imagem?
![image](https://user-images.githubusercontent.com/84328491/123308253-59103600-d51b-11eb-941a-1f47bc1f9b67.png)
Cumprimentos

**Comment by joaomlourenco** [comment, marked as answer]
In file “Config/8_listof.tex”, replace lines
```latex
\netaddlistof{listoffigures}      % The List of Figures. Comment to omit.
\netaddlistof{listoftables}       % The List of Tables. Comment to omit.
```
with
```latex
\netaddlistof{listoffigures*}      % The List of Figures. Comment to omit.
\netaddlistof{listoftables*}       % The List of Tables. Comment to omit.
```
but please note that this will also remove these lists from the PDF bookmarks.

**Reply by nunes3000** [reply]
Hello João
Thank you for the answer.
There is no other way to remove the list of figures and tables from the list of contents without remove these lists from the PDF bookmarks?

**Reply by b-pereira** [reply]
You can add the bookmarks manually, being careful to changes in the pagination by using the `bookmarks` package. Check [PDF Bookmarks with LaTeX ](https://michaelgoerz.net/notes/pdf-bookmarks-with-latex.html) for a demo of bookmarks inclusion per level and page. There are other ways to do that by name (like `\pdfbookmark{\contentsname}{Contents}`, but the way the template is structured doesn't allow it and it's ill-advised to do so. 

I didn't test it myself. So, if it works, please leave a a reply!
